pkg/kube/translation/apisix_route.go

```diff
@@ -329,8 +389,9 @@ func (t *translator) translateTCPRoute(ctx *TranslateContext, ar *configv2alpha1
 		if err != nil {
 			return err
 		}
-		sr.Upstream = ups
+		sr.UpstreamId = ups.ID
 		ctx.addStreamRoute(sr)
+		ctx.addUpstream(ups)
```

Comment:
Should check whether it existed already.

Reply:
Done.